### PR TITLE
Tensor.where method

### PR DESCRIPTION
**Pull request by python273**

### CODE REVIEW

File: test/test_ops.py [hunk 1]
```diff
@@ -66,13 +66,13 @@ def test_where(self):
     helper_test_op(
       [(100,)],
       lambda x: torch.where(x > 0.5, 4, 2),
-      lambda x: Tensor.where(x > 0.5, 4, 2), forward_only=True)
+      lambda x: (x > 0.5).where(4, 2), forward_only=True)
```
Comment (geohot) [inline test/test_ops.py:69]: The key of removing the staticmethod is both version should work.

Comment (python273) [reply test/test_ops.py:69]: yea, both versions work. Do you want an explicit test for `Tensor.where(x>0.5, 4, 2)`?

Comment (python273) [reply test/test_ops.py:69]: linter doesn't like non-`self` first arg, if that's what you mean

Comment (geohot) [reply test/test_ops.py:69]: Eh, it's all good.

**Pull request merged**